src/reporter.js

```diff
@@ -64,8 +62,7 @@ module.exports = async (answers) => {
   console.log("");
   console.log("Be patient, this may take a bit...");
 
-  spinner = new Spinner("Fetching results...");
```

Comment:
Is this to make it more clean? I liked the spinner because it showed it was thinking and working. No strong opinion either way

Reply:
I liked the spinner too but this is too keep a better log.